src/pages/patientView/genomicOverview/Tracks.tsx

```diff
@@ -112,10 +114,24 @@ export default class Tracks extends React.Component<TracksPropTypes, {}> {
 
     }
 
+    private svgContainer: any;
```

Comment:
it's a div so, you can type this as HTMLDivElement.    try to avoid using any.  it's not that big a deal here but good practice in general

Reply:
Thanks, Aaron. I just updated this. And I also changed the position of the download button.